katas/content/complex_arithmetic/complex_addition/Verification.qs

```diff
@@ -0,0 +1,47 @@
+namespace Kata.Verification {
+    open Microsoft.Quantum.Math;
+    open Microsoft.Quantum.Random;    
+    open Microsoft.Quantum.Convert; 
+
+    operation ComplexAdd_Reference(x : Complex, y : Complex) : Complex {
+    
+        // Calculate the sum of two complex numbers.  
+        let z = Complex (x::Real + y::Real, x::Imag + y::Imag);
+        return z;
+        }
+
+    @EntryPoint()
+    operation CheckSolution() : Bool {
+        
+            mutable success = false;
```

Comment:
Verification logic, here and in other tasks, can be made a lot shorter and more readable by using a for loop. Q# doesn't have `break` but you can return mid-for loop (if you're not using qubits) if a test fails. Loop for 25 iterations, use immutable variables for expected and actual, if they're different, print the fail message and return false, and outside the loop only print Correct! and return true